Tests/SwiftGRPCTests/MetadataTests.swift

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Foundation
+@testable import SwiftGRPC
+import XCTest
+
+class MetadataTests: XCTestCase {
+  static var allTests: [(String, (MetadataTests) -> () throws -> Void)] {
+    return [
+      ("testCopyDoesNotCorruptData", testCopyDoesNotCorruptData)
+    ]
+  }
+  
+  func testCopyDoesNotCorruptData() {
+    let metadata = try! Metadata(["foo": "bar"])
```

Comment:
Same comment here regarding `try`/`throws`

Reply:
See below

Reply:
This one was referring to marking the function as `throws` instead of using `try!`

Reply:
FWIW I actually prefer `try!` in tests; it gives more descriptive error messages. And given that the default case should be a pass, anyway, this should have little practical impact.